### PR TITLE
docs/variants/tuxedo_ibs15/: add platform hardware matrix

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemysław Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/variants/tuxedo_ibs15/testing-configuration.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Wireless card: Intel Wi-Fi 6 AX201
+- Monitor: HDMI 1920x1080p
+- USB:
+  + 2x USB SanDisk 3.2Gen1 16 GB
+  + Kingston DataTraveler 3.1Gen1 16 GB
+- USB Keyboard: DELL KB216
+- SD Card: SanDisk MicroSDHC 16 GB
+- Headset: Sony MDR-EX15AP
+
+## InfinityBook S 15 #1
```
Comment (mkopec) [inline docs/variants/tuxedo_ibs15/testing-configuration.md:25]:
```suggestion
## InfinityBook S 15 #2
```

File: docs/variants/tuxedo_ibs15/testing-configuration.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Hardware configuration matrix
+
+## Introduction
+
+This document describes the hardware configurations used for validation of the
+coreboot port.
+
+## InfinityBook S 15 #1
+
+- SSD:
```
Comment (mkopec) [inline docs/variants/tuxedo_ibs15/testing-configuration.md:10]:
```suggestion
- CPU: Intel Core i5-1135G7 
- SSD:
```

File: docs/variants/tuxedo_ibs15/testing-configuration.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- USB:
+  + 2x USB SanDisk 3.2Gen1 16 GB
+  + Kingston DataTraveler 3.1Gen1 16 GB
+- USB Keyboard: DELL KB216
+- SD Card: SanDisk MicroSDHC 16 GB
+- Headset: Sony MDR-EX15AP
+
+## InfinityBook S 15 #1
+
+- SSD:
```
Comment (mkopec) [inline docs/variants/tuxedo_ibs15/testing-configuration.md:27]:
```suggestion
- CPU: Intel Core i5-1135G7
- SSD:
```

File: docs/variants/tuxedo_ibs15/testing-configuration.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/tuxedo_ibs15/testing-configuration.md:25]: Improved

File: docs/variants/tuxedo_ibs15/testing-configuration.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/tuxedo_ibs15/testing-configuration.md:10]: Information about CPU has been added

File: docs/variants/tuxedo_ibs15/testing-configuration.md (hunk 3 above)
Comment (sulewskiprzemyslaw) [reply docs/variants/tuxedo_ibs15/testing-configuration.md:27]: Information about CPU has been added

**Pull request merged**